docker-compose.dev.yml

```diff
@@ -33,10 +32,27 @@ services:
 
   db:
     image: postgres:12.5
+    stdin_open: true
+    tty: true
     volumes:
       # isolate and persist dev database.
+      - db-data:/var/lib/postgresql/data
       - project-data:/project
+
+  pgadmin:
```

Comment:
with the additions here and in the unit test yml files are you removing the composition with docker-compose.yml or is something else going on?

Reply:
I really like it the pgadmin!  Its just in here to make things easier to map together with the db volume.